src/Packetery/Module/Order/LabelPrint.php

```diff
@@ -355,6 +358,9 @@ private function requestCarrierLabels( int $offset, array $packetIds ): Response
 		if ( ! $response->hasFault() ) {
 			foreach ( array_keys( $packetIdsWithCourierNumbers ) as $orderId ) {
 				$order = $this->orderRepository->getById( $orderId );
+				if ( null === $order ) { // Will not happen, collection was already limited.
```

Comment:
Mate mě ten komentář. Pokud je podmínka zbytečná, tak jí prosím odstraň. A nebo odstraň/aktualizuj komentář.

Reply:
Je potřeba ošetřit, že getById může vrátit null. Komentář jsem upravil.